### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
-# devextreme-rwa-demos
-DevExtreme RWA Demos
+# Devextreme UI Template Gallery
+
+Devextreme UI Template Gallery is a repository with a real-word application demo. This demo illustrates how to integrate various DevExtreme UI components into a web application. The projects in this repository includes the CRM and Project Management UI/UX patterns with source code for Angular.
```
Comment (dmitry-eliseev-devexpress) [inline README.md:3]: The projects include, not includes.

```suggestion
Devextreme UI Template Gallery is a repository with a real-word application demo. This demo illustrates how to integrate various DevExtreme UI components into a web application. The projects in this repository include the CRM and Project Management UI/UX patterns with source code for Angular.
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]
Typos.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+
+    ```
+    cd packages/angular
+
+    npm run start
+    ```
+
+3. Navigate to [http://localhost:4200/](http://localhost:4200/). 
+    
+    The project in default mode includes the navigation menu at the left. To navigate between the CRM and Planning patterns in embedded mode, use the following links:
```
Comment (dirk-pieterse) [inline README.md:35]:
```suggestion
    The project shows the navigation menu on the left in default mode. To navigate between CRM and Planning patterns in embedded mode, use the following links:
```

**Review by dirk-pieterse — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+
+The UI Templates have responsive layouts with DevExtreme UI components. Note that the components use sample data entities without any business logic. You can use these UI Templates in your project and adapt them to your specific business requirements.
+
+> The repository is in preview mode. Only Angular version is available for use. Vue and React projects are still under development.
+
+![Devextreme UI Template Gallery](/images/ui-template-gallery.png)
+
+## How It Organized
+
+UI Template sources for Angular are located in the [packages/angular](packages/angular) folder. Navigate to the following folders to get code for each template:
```
Comment (dirk-pieterse) [inline README.md:18]:
```suggestion
UI Template sources for Angular are in the [packages/angular](packages/angular) folder. Navigate to the following folders to get the code for each template:
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**